timely/src/dataflow/stream.rs

```diff
@@ -10,15 +10,14 @@ use crate::communication::Push;
 use crate::dataflow::Scope;
 use crate::dataflow::channels::pushers::tee::TeeHelper;
 use crate::dataflow::channels::Bundle;
-use std::fmt;
 
 // use dataflow::scopes::root::loggers::CHANNELS_Q;
 
 /// Abstraction of a stream of `D: Data` records timestamped with `S::Timestamp`.
 ///
 /// Internally `Stream` maintains a list of data recipients who should be presented with data
 /// produced by the source of the stream.
-#[derive(Clone)]
+#[derive(Debug, Clone)]
```

Comment:
First time I'm noticing this, but I think this will only allow `Debug` when `S` and `D` implement `Debug`. Is that intended? (vs the manual impl being removed that implements `Debug` without debug constraints.

Reply:
Good catch, I believe you're correct so I'll revert that
=